src/components/CanvasLayers.js

```diff
@@ -30,11 +30,11 @@ const reorder = (list, startIndex, endIndex) => {
 /** */
 export class CanvasLayers extends Component {
   /** */
-  static getUseableLabel(resource, index) {
+  static getUseableLabel(resource, index, langs) {
```

Comment:
This probably goes for all these label helpers... I wonder if we should refactor these back into the instance, now that they depend on some state data?

Reply:
Probably a good idea, yes! Or maybe refactor it into a utility function? I've seen this exact static method (and minor variations of it) in more than a handful components.